### PR TITLE
Cargo format run

### PR DESCRIPTION
**Pull request by kjeremy**
I have another branch where format changes muddled the diff so figured I'd submit a `cargo format` run.

### CODE REVIEW

File: crates/ra_assists/src/add_explicit_type.rs [hunk 1]
```diff
@@ -1,10 +1,7 @@
-use hir::{
-    HirDisplay, Ty,
-    db::HirDatabase,
-};
+use hir::{HirDisplay, Ty, db::HirDatabase};
```
Comment (matklad) [inline crates/ra_assists/src/add_explicit_type.rs]: Hm, we explicitelly disable import reordeting via `rustfmt.toml`:

https://github.com/rust-analyzer/rust-analyzer/blob/4e8664d9d37021e85d72b6228fa45e7edf4a1a74/rustfmt.toml#L1

why do we have this changes?

**Comment by lnicola** [comment]
That looks like imports_layout? There's no reordering involved.

**Comment by matklad** [comment]
aha, makes sense. I've re-enabled import reordeting in https://github.com/rust-analyzer/rust-analyzer/pull/1486 as well, closing in favor of that.

File: crates/ra_assists/src/add_explicit_type.rs (hunk 1 above)
Comment (kjeremy) [reply crates/ra_assists/src/add_explicit_type.rs]: Not sure. I didn't change the `rustfmt.toml` file. Maybe that's `import_layouts` or `merge_imports`?

**Pull request closed without merging**